wikidata/multilingual.py

```diff
@@ -3,39 +3,42 @@
 
 """
 import collections.abc
-from typing import Iterator, Mapping, Type, Union
+from typing import Iterator, Mapping, NewType, Type, Union
 
-from babel.core import Locale
+__all__ = 'MonolingualText', 'MultilingualText'
```

Comment:
`Locale` should be exposed as a public API so that type checkers can infer its inner type.  It also should be recorded as an added type in the changelog.

```suggestion
__all__ = 'Locale', 'MonolingualText', 'MultilingualText'
```